enterprise/frontend/src/embedding-sdk/components/public/CreateQuestion/CreateQuestion.tsx

```diff
@@ -1,23 +1,115 @@
-import { QuestionEditor } from "embedding-sdk/components/private/QuestionEditor";
+import { useState } from "react";
 
-import type { InteractiveQuestionProps } from "../InteractiveQuestion";
+import { FlexibleSizeComponent } from "embedding-sdk";
+import { useInteractiveQuestionContext } from "embedding-sdk/components/private/InteractiveQuestion/context";
+import { SaveQuestionModal } from "metabase/containers/SaveQuestionModal";
+import { Button, Flex } from "metabase/ui";
 
-type CreateQuestionProps = Omit<InteractiveQuestionProps, "questionId">;
+import {
+  InteractiveQuestion,
+  type InteractiveQuestionProps,
+} from "../InteractiveQuestion";
+
+type CreateQuestionProps = Partial<
+  Omit<InteractiveQuestionProps, "questionId" | "children">
+>;
 
 export const CreateQuestion = ({
-  plugins,
   onSave,
-  onBeforeSave,
-  entityTypeFilter,
-  isSaveEnabled,
-  saveToCollectionId,
-}: CreateQuestionProps = {}) => (
-  <QuestionEditor
-    plugins={plugins}
-    onBeforeSave={onBeforeSave}
-    onSave={onSave}
-    entityTypeFilter={entityTypeFilter}
-    isSaveEnabled={isSaveEnabled}
-    saveToCollectionId={saveToCollectionId}
-  />
-);
+  isSaveEnabled = true,
+  ...props
+}: CreateQuestionProps = {}) => {
+  const [isSaveModalOpen, setSaveModalOpen] = useState(false);
+
+  return (
+    <InteractiveQuestion
+      {...props}
+      isSaveEnabled={isSaveEnabled}
```

Comment:
The save button is hidden when the user passes `isSaveEnabled={false}`, mostly useful for demonstration purposes (e.g. allowing to create a question to query data, but not to save them).